prow/getting_started.md

```diff
@@ -0,0 +1,81 @@
+# How to turn up a new cluster
+
+Prow should run anywhere that Kubernetes runs. Here are the steps required to
+set up a basic prow cluster on [GKE](https://cloud.google.com/container-engine/).
+You'll need a GCP project. Prow will work on any Kubernetes cluster, however,
+so feel free to turn up a cluster some other way and skip the first step.
+
+## Create the cluster.
+
+I'm assuming that `PROJECT` and `ZONE` environment variables are set. Run the
+following to create the cluster. This will also set up `kubectl` to point to
+the new cluster.
+
+```sh
+gcloud -q container --project "${PROJECT}" clusters create prow \
+  --zone "${ZONE}" --machine-type n1-standard-4 --num-nodes 2
+```
+
+## Create the GitHub secrets.
+
+You will need two secrets to talk to GitHub. The `hmac-token` is the token that
+you give to GitHub for validating webhooks. Generate it using any reasonable
+randomness-generator. The `oauth-token` is an OAuth2 token that has read and
+write access to the bot account. Generate it from the account's settings ->
+Personal access tokens -> Generate new token.
+
+```sh
+kubectl create secret generic hmac-token --from-file=hmac=/path/to/hook/secret
+kubectl create secret generic oauth-token --from-file=oauth=/path/to/oauth/secret
+```
+
+## Run the prow components in the cluster.
+
+```sh
+kubectl apply -f cluster/starter.yaml
+```
+
+After a moment, the cluster components will be running.
+
+```sh
+$ kubectl get deployments
+NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deck         2         2         2            2           1m
+hook         2         2         2            2           1m
+horologium   1         1         1            1           1m
+plank        1         1         1            1           1m
+sinker       1         1         1            1           1m
+```
+
+Find out your external address. It might take a couple minutes for the IP to
+show up.
+
+```sh
+$ kubectl get ingress ing
+NAME      HOSTS     ADDRESS          PORTS     AGE
+ing       *         an.ip.addr.ess   80        3m
+```
+
+Go to that address and verify that the "echo-test" job has a green check-mark
```

Comment:
Open the address in your web browser? Include a screenshot of what this green check mark should look like?

Reply:
It looks like a green check mark.